websites/S/Sonic Retro/presence.ts

```diff
@@ -0,0 +1,279 @@
+const presence = new Presence({
+		clientId: "970743721404530798"
+	}),
+	browsingTimestamp = Math.floor(Date.now());
```

Comment:
you're flooring an integer, you probably wanna do this instead
```suggestion
	browsingTimestamp = Math.floor(Date.now() / 1000);
```

Reply:
> you're flooring an integer, you probably wanna do this instead
> 
> ```suggestion
> 
> 	browsingTimestamp = Math.floor(Date.now() / 1000);
> 
> ```

My bad, used an old presence as a template and forgot to change it 😅